### PR TITLE
Fix action is_present field not being reset

### PR DESCRIPTION
**Pull request by erlingrj**
See https://github.com/lf-lang/lingua-franca/pull/2409

### CODE REVIEW

File: core/reactor_common.c [hunk 1]
```diff
@@ -1096,6 +1103,13 @@ void initialize_global(void) {
   // Call the code-generated function to initialize all actions, timers, and ports
   // This is done for all environments/enclaves at the same time.
   _lf_initialize_trigger_objects();
+
+#if !defined(LF_SINGLE_THREADED) && !defined(NDBEUG)
```
Comment (edwardalee) [inline core/reactor_common.c:1107]:
```suggestion
#if !defined(LF_SINGLE_THREADED) && !defined(NDEBUG)
```

**Review by edwardalee — APPROVED** [review]
LGTM!  I'll look at the lingua-franca PR next.

**Comment by erlingrj** [comment]
@edwardalee there is still some issue with that I am debugging. I can ping you when it is resolved!

**Comment by edwardalee** [comment]
> @edwardalee there is still some issue with that I am debugging. I can ping you when it is resolved!

Looking more closely at this:

```
temp.pr(
    enclaveStruct
        + ".is_present_fields["
        + CUtil.bankIndex(instance)
        + " + "
        + enclaveInfo.numIsPresentFields
        + " + count] = &"
        + portRef
        + con
        + "is_present;");
```

I think maybe instead of `bankIndex(instance)` it should be `bankIndex(port.getParent())`.

**Pull request merged**